docs/en/platform-engineers/keda.md

```diff
@@ -80,15 +80,15 @@ schematic:
       			name: context.name
       		}
       		triggers: [{
-      			type: paramter.type
+      			type: parameter.type
       			metadata: {
       				type:  "Utilization"
-      				value: paramter.value
+      				value: parameter.value
       			}
       		}]
       	}
       }
-      paramter: {
+      parameter: {
```

Comment:
does it have to be called "parameter" or it's just a typo fix

Reply:
It have to be called "parameter",  otherwise vela can't automatically generate OpenAPI v3 JSON schema based on its parameter list.